<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="rasta" name="com.myrbshoetests.tests.ShoePriceAvailableTest" tests="3" failures="1" timestamp="19 Jun 2017 19:50:14 GMT" time="38.331" errors="0">
  <testcase name="isShoePriceAvailable" time="9.796" classname="com.myrbshoetests.tests.ShoePriceAvailableTest"/>
  <testcase name="isShoePriceAvailable" time="6.800" classname="com.myrbshoetests.tests.ShoePriceAvailableTest"/>
  <testcase name="isShoePriceAvailable" time="21.735" classname="com.myrbshoetests.tests.ShoePriceAvailableTest">
    <failure type="java.lang.AssertionError" message="The shoe price is not available. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The shoe price is not available. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.myrbshoetests.tests.ShoePriceAvailableTest.isShoePriceAvailable(ShoePriceAvailableTest.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- isShoePriceAvailable -->
</testsuite> <!-- com.myrbshoetests.tests.ShoePriceAvailableTest -->
